terraform/custom_domains/environmental_domains/workspace_variables/afqts_test_backend.tfvars

```
@@ -0,0 +1,4 @@
+resource_group_name  = "s189p01-afqtsdomains-rg"
+storage_account_name = "s189p01afqtsdomainstf"
```

```suggestion
storage_account_name = "s189p01afqtsdomainstfsa"
```

I believe the convention is to suffix storage account names with `sa`.

---

This storage account has already been deployed, and would require a new ticket to change.